dockerRunner.go

```diff
@@ -1087,3 +858,232 @@ func (dr *dockerRunnerImpl) DeleteBridgeNetwork(ctx context.Context) error {
 
 	return nil
 }
+
+func (dr *dockerRunnerImpl) generateEntrypointScript(shell string, commands []string, commandsNotAsJob bool) (path string, extension string, err error) {
+
+	r, _ := regexp.Compile("[a-zA-Z0-9_]+=|;|\\||&&|\\|\\|")
```

Comment:
Does this allow spaces between variable name, equal sign and value?

Reply:
Nope, but that's not valid in bash anyway to assign variables.

Reply:
oh yeah, forgot about that